tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -1020,9 +1020,9 @@ public void TestGroupBy()
         [Fact]
         public void TestGoupByDifferentColumnTypes()
         {
-            void GroupCountAndAssert(DataFrame df)
+            void GroupCountAndAssert(DataFrame frame)
```

Comment:
Lol. Unrelated parameter renames got snuck into this PR :)

Reply:
These changes were related. Before, this project didn’t have warnings as errors turned on. With directory build props, now it is on for all projects. These places were causing errors because the names conflicted with variables in the outer scope.

Reply:
Ah! I didn't notice that. `string data` used twice within the same method in different scopes. Nice. Thanks for clarifying.